### PR TITLE
feat: elys spec add

### PR DESCRIPTION
**Pull request by BabyScope**
Spec sheet for Elys Testnet 

[Portal log Elys.txt](https://github.com/lavanet/lava/files/15204698/Portal.log.Elys.txt)
[Provider log Elys.txt](https://github.com/lavanet/lava/files/15204700/Provider.log.Elys.txt)

### CODE REVIEW

File: cookbook/specs/spec_add_elys.json [hunk 1]
```diff
@@ -0,0 +1,2964 @@
...
+                                "category": {
+                                    "deterministic": true,
+                                    "local": false,
+                                    "subscription": false,
+                                    "stateful": 0
+                                },
+                                "extra_compute_units": 0
+                            },
+                            {
+                                "name": "/elys-network/elys/accountedpool/accounted_pool/{ppol_id}",
```
Comment (oren-lava) [inline cookbook/specs/spec_add_elys.json:53]: typo: ppol -> pool

File: cookbook/specs/spec_add_elys.json [hunk 2]
```diff
@@ -0,0 +1,2964 @@
...
+                    },
+                    {
+                        "enabled": true,
+                        "collection_data": {
+                            "api_interface": "grpc",
+                            "internal_path": "",
+                            "type": "",
+                            "add_on": ""
+                        },
+                        "apis": [
```
Comment (oren-lava) [inline cookbook/specs/spec_add_elys.json:1406]: add grpc.reflection.v1alpha.ServerReflection/ServerReflectionInfo

**Review by oren-lava — CHANGES_REQUESTED** [review]

File: cookbook/specs/spec_add_elys.json (hunk 2 above)
Comment (oren-lava) [reply cookbook/specs/spec_add_elys.json:1406]: not relevant. found in COSMOSSDK

File: cookbook/specs/spec_add_elys.json [hunk 3]
```diff
@@ -0,0 +1,2964 @@
...
+                                    {
+                                        "expected_value": "elystestnet-1"
+                                    }
+                                ]
+                            },
+                            {
+                                "name": "pruning",
+                                "values": [
+                                    {
+                                        "latest_distance": 0
```
[the remark this answers is not in the document]
Comment (BabyScope) [reply cookbook/specs/spec_add_elys.json]: This was for my test i made a local node to test it out pruning did not reach it.

But fixed it with this 

https://github.com/lavanet/lava/pull/1407/commits/a8320fbf46d0904a46ddde41a9574eece3656248

File: cookbook/specs/spec_add_elys.json (hunk 1 above)
Comment (BabyScope) [reply cookbook/specs/spec_add_elys.json:53]: Fixed 

https://github.com/lavanet/lava/pull/1407/commits/a8320fbf46d0904a46ddde41a9574eece3656248

Comment (oren-lava) [reply cookbook/specs/spec_add_elys.json:53]: I don't see you fixed it

File: cookbook/specs/elys.json [hunk 4]
```diff
@@ -0,0 +1,2964 @@
+{
+    "proposal": {
+        "title": "Add Specs: EYLS",
```
Comment (Yaroms) [inline cookbook/specs/elys.json:3]: misspelling

File: cookbook/specs/elys.json [hunk 5]
```diff
@@ -0,0 +1,2964 @@
...
+        "description": "Adding new specification support for relaying eyls data on Lava",
+        "specs": [
+            {
+                "index": "EYLS",
```
Comment (Yaroms) [inline cookbook/specs/elys.json:7]: misspelling

File: cookbook/specs/elys.json [hunk 6]
```diff
@@ -0,0 +1,2964 @@
+{
+    "proposal": {
+        "title": "Add Specs: EYLS",
+        "description": "Adding new specification support for relaying eyls data on Lava",
```
Comment (Yaroms) [inline cookbook/specs/elys.json:4]: misspelling

File: cookbook/specs/elys.json [hunk 7]
```diff
@@ -0,0 +1,2964 @@
...
+                                "name": "chain-id",
+                                "values": [
+                                    {
+                                        "expected_value": "elystestnet-1"
+                                    }
+                                ]
+                            },
+                            {
+                                "name": "pruning",
+                                "values": [
```
Comment (Yaroms) [inline cookbook/specs/elys.json:1381]: we miss archive verification here

File: cookbook/specs/elys.json [hunk 8]
```diff
@@ -0,0 +1,2964 @@
...
+    "proposal": {
+        "title": "Add Specs: EYLS",
+        "description": "Adding new specification support for relaying eyls data on Lava",
+        "specs": [
+            {
+                "index": "EYLS",
+                "name": "eyls testnet",
+                "enabled": true,
+                "imports": [
+                    "COSMOSSDKFULL"
```
Comment (Yaroms) [inline cookbook/specs/elys.json:11]: does elys support cosmwasm?
COSMOSSDKFULL -> with cosmwasm
COSMOSSDK -> without cosmwasm

Comment (BabyScope) [reply cookbook/specs/elys.json:11]: image
yes they support wasm

File: cookbook/specs/elys.json (hunk 7 above)
Comment (BabyScope) [reply cookbook/specs/elys.json:1381]: added the archive

File: cookbook/specs/elys.json [hunk 9]
```diff
@@ -0,0 +1,2964 @@
+{
+    "proposal": {
+        "title": "Add Specs: EYLS",
+        "description": "Adding new specification support for relaying eyls data on Lava",
+        "specs": [
+            {
+                "index": "EYLS",
+                "name": "eyls testnet",
```
[the remark this answers is not in the document]
Comment (BabyScope) [reply cookbook/specs/elys.json]: corrected

File: cookbook/specs/elys.json (hunk 6 above)
Comment (BabyScope) [reply cookbook/specs/elys.json]: corrected

File: cookbook/specs/elys.json (hunk 5 above)
Comment (BabyScope) [reply cookbook/specs/elys.json]: corrected

File: cookbook/specs/elys.json (hunk 4 above)
Comment (BabyScope) [reply cookbook/specs/elys.json]: Resolved in newest update

File: cookbook/specs/elys.json [hunk 10]
```diff
@@ -0,0 +1,2972 @@
...
+                                    }
+                                ]
+                            }
+                        ],
+                        "extensions": [
+                            {
+                                "name": "archive",
+                                "cu_multiplier": 5,
+                                "rule": {
+                                    "block": 19200
```
Comment (omerlavanet) [inline cookbook/specs/elys.json:2962]: route rule should be lower than the verification to make sure there's no possible for race when there's a delay in chain fetcher or staggering providers

**Review by omerlavanet — DISMISSED** [review]

**Review by Yaroms — APPROVED** [review]

**Pull request merged**